public/services/CampaignsApi.js

```diff
@@ -31,16 +31,16 @@ export function deleteCampaign(id) {
   });
 }
 
-export function fetchLatestAnalytics() {
+export function fetchLatestAnalytics(territory) {
   return AuthedReqwest({
-    url: '/api/v2/campaigns/latestAnalytics',
+    url: '/api/v2/campaigns/latestAnalytics?territory=' + territory,
     method: 'get'
   });
 }
 
-export function fetchLatestAnalyticsForCampaign(id) {
+export function fetchLatestAnalyticsForCampaign(id, territory) {
   return AuthedReqwest({
-    url: '/api/v2/campaigns/' + id + '/latestAnalytics',
+    url: '/api/v2/campaigns/' + id + '/latestAnalytics?territory=' + territory,
```

Comment:
You can use interpolation, eg. `campaigns/&{id}/...?territory=${territory}`

Reply:
Good point - will change.